tools/dep_updaters/update-biome.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# Shell script to update Biome in the source tree to the latest release.
+
+# This script must be in the tools directory when it runs because it uses the
+# script source file path to determine directories to work in.
+
+set -ex
+
+ROOT=$(cd "$(dirname "$0")/../.." && pwd)
+
+[ -z "$NODE" ] && NODE="$ROOT/out/Release/node"
+[ -x "$NODE" ] || NODE=$(command -v node)
+NPM="$ROOT/deps/npm/bin/npm-cli.js"
+
+# shellcheck disable=SC1091
+. "$ROOT/tools/dep_updaters/utils.sh"
+
+NEW_VERSION=$("$NODE" "$NPM" view @biomejs/biome dist-tags.latest)
+CURRENT_VERSION=$("$NODE" -p "require('./tools/node_modules/@biomejs/biome/package.json').version")
+
+# This function exit with 0 if new version and current version are the same
+compare_dependency_version "@biomejs/biome" "$NEW_VERSION" "$CURRENT_VERSION"
+
+cd "$( dirname "$0" )" || exit
+rm -rf ../node_modules/@biomejs/biome
+(
```

Comment:
we are changing the installation to `npm pack` from `npm i`
For reference: https://github.com/nodejs/node/blob/main/tools/dep_updaters/update-acorn-walk.sh